### PR TITLE
adding lightbox2 override

### PR DESCRIPTION
**Pull request by AshleyGrant**

### CODE REVIEW

File: package-overrides/github/lightbox2/lightbox2@2.7.1.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "main": "js/lightbox",
+  "dependencies": {
+    "jquery": "^2.1.3",
+    "css": "0.1.0"
+  },
+  "shim": {
+    "js/lightbox": {
+      "format": "global",
```
Comment (guybedford) [inline package-overrides/github/lightbox2/lightbox2@2.7.1.json]: This should be at the base-level of the package.json.

**Comment by guybedford** [comment]
Sorry for the delay on this, let me know if the correction is possible and we can merge.

**Comment by AshleyGrant** [comment]
I believe I've fixed what you were asking me to change by moving "format": "global" to the root.

**Comment by guybedford** [comment]
Thanks, I'm away till Saturday - hopefully someone else can help with
merging in the mean time.

**Comment by computmaxer** [comment]
Looks good, thanks

**Pull request merged**